.github/workflows/label-check.yml

```diff
@@ -4,8 +4,10 @@ on:
   pull_request:
     types:
       - opened
+      - repoened
       - labeled
       - unlabeled
+      - synchronize
```

Comment:
@jarrodmillman note I added this to the event for the label check -- otherwise it only runs on the first commit and no subsequent ones. I think we want it to run for all so we can add it to the requirements for merge-when-green

Reply:
(you can see it ran on 12e59fd where I added this to the event list but not on the preceding commit f18e780)